### PR TITLE
[jsk_pr2_startup] change realtime_socket_timeout and max_pd_retries

### PR DESCRIPTION
**Pull request by knorth55**
change the `realtime_socket_timeout` and `max_pd_retries` to `20000` and `5`.
PR2 realtime loop checks the ethercat loop with timeout 20ms and retries 5 times.
PR2 stops its servo when ethercat connection return is slower than `20*5=100ms`,
which is the same as previous timeout.

detailed info: http://wiki.ros.org/ethercat_hardware/Tutorials/ChangingRealtimePacketTimeout
cc. @ishiguroJSK

### CODE REVIEW

**Comment by knorth55** [comment]
please read here for more detailed information of pr2 realtime control.

https://github.com/PR2/pr2_ethercat_drivers/blob/kinetic-devel/ethercat_hardware/src/ethercat_hardware.cpp#L295-L340
https://github.com/PR2/pr2_robot/blob/kinetic-devel/pr2_ethercat/src/main.cpp#L289-L300

File: jsk_pr2_robot/jsk_pr2_startup/pr2_bringup.launch [hunk 1]
```diff
@@ -18,7 +18,8 @@
 
   <!-- pr2_etherCAT -->
   <node name="realtime_loop" machine="c1" launch-prefix="pr2-grant" pkg="pr2_ethercat" type="pr2_ethercat" args="-i ecat0 -x robot_description" output="screen" />
-  <param name="pr2_etherCAT/realtime_socket_timeout" value="5000" />
+  <param name="pr2_ethercat/realtime_socket_timeout" value="20000" />
```
Comment (iory) [inline jsk_pr2_robot/jsk_pr2_startup/pr2_bringup.launch:21]: Please add a comment to describe this change.

**Review by iory — CHANGES_REQUESTED** [review]

File: jsk_pr2_robot/jsk_pr2_startup/pr2_bringup.launch (hunk 1 above)
Comment (knorth55) [reply jsk_pr2_robot/jsk_pr2_startup/pr2_bringup.launch:24]: thanks, i add comment

**Review by iory — APPROVED** [review]

**Review by k-okada — COMMENTED** [review]
@knorth55 sorry could you resolve conflicts?

**Comment by knorth55** [comment]
I resolve the conflicts.

**Pull request merged**